src/insights_client/tests/test_motd.py

```diff
@@ -0,0 +1,116 @@
+import os
+import pathlib
+import tempfile
+import typing
+import unittest
+import unittest.mock
+
+import pytest
+
+import insights_client
+
+
+class MockFS:
+    """Mock filesystem.
+
+    Ensures that the files required for the MOTD manipulation are created
+    in a temporary directory instead of working with actual system files.
+    """
+
+    _dir: tempfile.TemporaryDirectory
+
+    def __init__(self):
+        self._dir = tempfile.TemporaryDirectory(prefix="test-chroot-")
+
+        paths: typing.Dict[str, str] = {
```

Comment:
It would make sense for this to be a dictionary, if it was a class attribute and it was then accessed to provide the actual paths based on the keys.